src/health.ts

```diff
@@ -0,0 +1,15 @@
+import * as express from 'express';
+
+let isHealthy = true;
+
+export const setIsHealthy = (value: boolean) => {
+  isHealthy = value;
+};
+
+const health = express();
+
+health.get('/', (_, res) => {
+  res.status(isHealthy ? 200 : 500).send();
+});
+
+export { health };
```

Comment:
I assume this is for health checks on the AWS platform?

Reply:
Yes, it's similar to App Engine.